### DISCUSSION TITLE
Unable to detect login share from the Auth Network.

### DISCUSSION THREAD

**Discussion by Felix-Asante** [Web3Auth/Web3Auth#689] [Help]
I am creating a custom login using web3auth core plug and play. I have integrated both google and facebook auth but whenever i try to authenticate with them, i get this error message: 
_**Unable to detect login share from the Auth Network. This may be due to slow internet connection. Check your internet speed and try again.**_

**Comment by himanshuchawla009** [comment]
Hello, it was an issue on ropsten network being deprecated, we have already fixed this. Can you try to do the login here: https://beta.openlogin.com/ 

Let me know if login on above link is not working for you. If it is working thn is should work with web3auth sdk as well. Please try to hard reload your page.

**Reply by Felix-Asante** [reply]
i was actually able you login with the link you provided but not on my app
these are the callbacks that i am using: https://app.openlogin.com/auth
https://beta.openlogin.com/auth

**Reply by himanshuchawla009** [reply]
Can you share console.logs of the page which is breaking from your app.

**Reply by Felix-Asante** [reply]
![Screen Shot 2022-10-06 at 8 28 30 PM](https://user-images.githubusercontent.com/62264989/194402192-c4e06349-d916-4660-a3c6-313f51d0dcc3.png)

here is a screenshoot from my console

**Comment by billyjacoby** [comment]
I'm seeing this exact same issue as well, is there anything else I can provide to help get this fixed?

**Reply by billyjacoby** [reply]
I'm thinking it may be an issue with custom domains via Auth0, I'm seeing this error:

```
customauth.esm.js:2316 Error: Error occurred while verifying paramscould not validate field iss, token https://auth-staging.DOMAIN.com/, expected https://DOMAIN.us.auth0.com/
    at torusUtils.esm.js:87:20
    at <anonymous>
```

**Reply by chaitanyapotti** [reply]
custom domains are not very easily supported with auth0

**Reply by billyjacoby** [reply]
Understood, how do I go about implementing it? Even if it isn't easy it's a use case that we need.
